apps/frontend/src/locales/en/translations.json

```diff
@@ -934,92 +934,6 @@
-                "cta": "Deposit stablecoins"
-            },
-            "social": {
-                "title": "Influence",
-                "description": "Earn points for your social influence! Like, Share, and Tweet to be given the POWA!",
-                "cta": "Twitter quests"
-            }
-        },
-        "tables": {
-            "connectWalletText": "Connect your wallet to see how many points you’ve got.",
-            "participantNotFound": "Take any action on the dapp to start earning points",
-            "connectWalletCta": "Connect wallet",
-            "baseTable": {
-                "participant": "Participant",
-                "points": "Points",
-                "yourPosition": "Your position",
-                "badges": {
-                    "earlyUser": "+5% Early user",
-                    "genesis": "+5% Genesis",
-                    "nft": "+5% NFT holder",
-                    "origin": "+5% Origin",
-                    "topImporterDay": "+10% Daily top importer",
-                    "topImporterWeek": "+10% Weekly top importer",
-                    "voter": "+5% Voter",
-                    "topStakerDay": "+10% Daily top staker",
-                    "topStakerWeek": "+10% Weekly top trader",
-                    "wallet": "+100 points"
-                }
-            },
-            "trading": {
-                "tabTitle": "Trading"
-            },
-            "staking": {
-                "tabTitle": "Staking",
-                "tableSubtitle": "Please note that the system is currently undercounting restaking. This will be addressed in a coming update."
-            },
-            "social": {
-                "tabTitle": "Influence"
-            }
-        },
-        "pointsSection": {
-            "title": "Gain POWA!",
-            "description": "Actions on the Sovryn dApp and on Twitter earn you points. Points convert to POWA - the next Bitcoin token class.",
-            "tables": {
-                "trading": {
-                    "title": "Trade",
-                    "item1": "Provide liquidity for SOV or DLLR AMMs, per $1 value",
-                    "item2": "Any AMM trade, per $1 value",
-                    "item3": "Import stablecoins, per $1 value",
-                    "item4": "Weekly top  importer",
-                    "item5": "Daily top importer"
-                },
-                "staking": {
-                    "title": "Stake",
-                    "item1": "New and re-stakes, 1VP",
-                    "item2": "Existing stakes, 1 VP",
-                    "item3": "Weekly top staker",
-                    "item4": "Daily top staker"
-                },
-                "social": {
-                    "title": "Influence",
-                    "item1": "See Influence leaderboard for current quests points breakdown."
-                }
-            }
-        }
-    },
     "leaderboardPointsPage": {
```

Comment:
pls remove `claimPowaPage`, `claimPowa` in the translation file as well since we don't need them anymore